src/universal/modules/userDashboard/containers/UserSettings/UserSettingsContainer.js

```diff
@@ -76,7 +89,56 @@ export default class UserSettingsContainer extends Component {
       router.push(nextPage);
     }
     untouch('preferredName');
-  };
+  }
+
+  uploadPicture(pictureFile) {
+    return cashay.mutate('createUserPicturePutUrl', {
+      variables: {
+        userFilename: pictureFile.name
```

Comment:
oh definitely, sorry this was one of those comments from yesterday (pre-refactor)